### PR TITLE
feat: Sentry 9 changelog

### PR DESCRIPTION
**Pull request by dcramer**

### CODE REVIEW

**Comment by dcramer** [comment]
Putting this up for initial review. I *think* I covered major things by skimming through all of the commits.

**Comment by dcramer** [comment]
Suspect commits should be available for VSTS and Bitbucket as well

**Comment by dcramer** [comment]
Updated owners description

File: CHANGES [hunk 1]
```diff
@@ -1,21 +1,116 @@
...
+  Expect-CT, and Expect-Staple reports.
+- Improved reprocessing feature.
+- Improved support for native Debug Symbols (Apple, Linux).
+- The ingestion API is now JSON-schema based (though still fairly flexible).
+- Added setup wizard for various SDKs (e.g. react-native).
+
+Privacy Changes
+~~~~~~~~~~~~~~~
+
+As part of GDPR, the open source version of Sentry now changes the default policy for it's beacon
```
Comment (JeanFred) [inline CHANGES]: it’s → its :)

File: CHANGES [hunk 2]
```diff
@@ -1,21 +1,114 @@
-Version 8.23 (Unreleased)
--------------------------
-- Experimental implementation of Slack actions via a new Integrations and Identity API.
-- Display the organization setting that was updated, along with the old/new value, in the Audit Log.
-- Group and ProjectGroupIndex endpoints now return AssignedTo as an object with the keys `id`, `type`, and `name`, instead of a full UserDetails object.
+Version 9.0
```
Comment (mattrobenolt) [inline CHANGES]: I'd only suggest bringing back `(Unreleased)` since we haven't tagged it yet.

**Review by mattrobenolt — APPROVED** [review]

**Pull request merged**